src/modules/core/components/Fields/Toggle/Toggle.tsx

```diff
@@ -18,7 +19,7 @@ interface Appearance {
 interface Props {
   appearance?: Appearance;
   name: string;
-  label?: string;
+  label?: string | MessageDescriptor;
```

Comment:
This should also have a `labelValues` which will bring in the (potentially) message descriptor variables